lib/main.js

```diff
@@ -391,9 +391,9 @@ const Hydrogen = {
 
   _runAll(kernel) {
     const breakpoints = this.codeManager.getBreakpoints();
-    _.forEach(breakpoints, (start, i) => {
-      if (i === breakpoints.length) return;
-      const end = breakpoints[i + 1];
+    _.forEach(breakpoints, (end, i) => {
+      if (i === 0) return;
+      const start = breakpoints[i - 1];
```

Comment:
I remember fixing this the same way in the past but I can't remember what the exact problem was.